projects/ngneat/transloco-locale/src/lib/transloco-locale.service.ts

```diff
@@ -62,6 +62,28 @@ export class TranslocoLocaleService implements OnDestroy {
     this._locale = locale;
   }
 
+  /**
+   * Get the currency symbol for the currently set locale.
+   */
+  getCurrencySymbol() {
```

Comment:
LGTM, I would just add an optional parameter for a locale something like this:
```ts
getCurrencySymbol(locale = this.getLocale()) {
```

Reply:
Yeah, good idea! Changed that

Reply:
Thanks, good work!